server/loginserver.ts

```diff
@@ -12,24 +12,22 @@
 const LOGIN_SERVER_TIMEOUT = 30000;
 const LOGIN_SERVER_BATCH_TIME = 1000;
 
+// tslint:disable-next-line no-var-requires
 const http = Config.loginserver.startsWith('http:') ? require("http") : require("https");
```

Comment:
`"http"` and `"https"` should be single quotes

(which wasn't introduced by this change, but lets fix it while we're here :P)

Reply:
Sure, lol.